libraries/AP_Baro/AP_Baro_MS5611.cpp

```diff
@@ -85,15 +85,21 @@ bool AP_Baro_MS56XX::_init()
         AP_HAL::panic("PANIC: AP_Baro_MS56XX: failed to take serial semaphore for init");
     }
 
+    // high retries for init
+    _dev->set_retries(10);
+    
     uint16_t prom[8];
     bool prom_read_ok = false;
-    
+
+    const char *name = "MS5611";
     switch (_ms56xx_type) {
     case BARO_MS5607:
+        name = "MS5607";
```

Comment:
wow, this works? As long as the length is the same I guess.. Which this is. Interesting!